recipes/sourmash/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
     - numpy
     - matplotlib
     - scipy
-    - khmer >=2.1,<3
+    - khmer >=2.1,<2.9
```

Comment:
I'd rather use `khmer >=2.1,<3.0a0` which is also the kind of constraint `conda-build` would generate when a dependency is pinned with a `max_pin`.